### DISCUSSION TITLE
[IntelliSense] Support `theme` in `tailwind.config.js`

### DISCUSSION THREAD

**Discussion by deadcoder0904** [tailwindlabs/tailwindcss#7552] [Ideas]
Not sure if it's possible yet but would love to have support for IntelliSense in `tailwind.config.js` when I use `@tailwindcss/typography`

I try to use it using `theme` where I have to type the whole thing which means more keystrokes to type.

Also, would love if `hover` shows values like when I get when I `hover` on `className` :)

I wrote what I do here -> https://github.com/tailwindlabs/blog.tailwindcss.com/issues/35

**Comment by dwelle** [comment]
Also the color previews and all the jazz!

![image](https://user-images.githubusercontent.com/5153846/149629830-67b6f193-754d-48cf-9de0-533457194a66.png)
